addons/velero/1.12.2/tmpl-troubleshoot.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kurl-velero-supportbundle-spec
+  labels:
+    troubleshoot.io/kind: support-bundle
```

Comment:
you're creating this in the default namespace

either include this in the files applied by kustomize, or include the namespace here
I would recommend including this in the files applied by kustomize

Reply:
Good point. I'll create the config maps in the same namespace as the addons.

Reply:
@laverya I made the requested change

Reply:
Almost good to approve - you need to add the kustomization change to the template files